### DISCUSSION TITLE
9.0.15无法以图片方式打开闪照

### DISCUSSION THREAD

**Discussion by AkinaHaruka** [cinit/QAuxiliary#789] [Q&A] [answered]
如题
![Screenshot_20240202_231111_com.tencent.mobileqq.jpg](https://github.com/cinit/QAuxiliary/assets/53249654/11d1be32-4f3f-4089-ac1c-b9a7809c87b6)

**Comment by HdShare** [comment, marked as answer]
尝试使用最新CI版
